README.md

```diff
@@ -6,8 +6,9 @@
 
 This package aims to provide a way to:
 
-- Embed Lua to any Node.js, Deno or Web Application.
-- Run lua code in any operational system
+- Embed Lua to any Node.js, Deno, or Web Application.
+- Run Lua code on any operating system
+- Run Lua programs in the browser (without JS)
```

Comment:
?

Reply:
Yes?

Reply:
"run Lua programs in the browser (without JS)" doesn't make sense, because JS is a essential component that glues LUA+DOM, and also wasmon is not a plug-in play like fengari-web, where you can do `<script type="application/lua">`